src/containers/career.jsx

```diff
@@ -114,16 +116,16 @@ class Career extends Component {
             onTouchEnd={event => this.handleTouchEnd(event)}>
             <div className="front">
               <CareerCard career={this.state.careers[this.state.currentIndex]}/>
-              <button
+              <Info size='2em'
                 onClick={() => {
                   this.flipCard();
-                }}> Info</button>
+                }}/>
             </div>
             <div className="back">
               <CareerInfo career={this.state.careers[this.state.currentIndex]}/>
-              <button className="btn1" onClick={() => {
+              <Back size='2em'className="btn1" onClick={() => {
```

Comment:
onClick on a new line.
also the => is a return you dont need the curly braces if you are returning 1 thing

Reply:
if I remove the curry braces, it says Uncaught ```Error: Module build failed: SyntaxError: Unexpected token, expected } (119:61)```